### PR TITLE
seed ability

### PR DESCRIPTION
**Pull request by gmd298**
# Relevant ticket
seed

# Changes made

* added seed ability

# Steps to QA

* run ~npm run db:seed~

### CODE REVIEW

**Comment by gmd298** [comment]
Hello Mr. Benton. I am requesting you review this.

File: scripts/seeds/seed.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+const chance = new Chance();
+
+const run = async () => {
+  logger.info('connecting to mongoose...');
+  await mongoose.connect(dbUrl);
+  logger.success('successfully connected to mongo');
+
+  logger.info('clearing database...');
+  // await Promise.all(mongoose.connection.collections.array.map((collection)
```
Comment (JakeBent) [inline scripts/seeds/seed.js:16]: rm comments

File: src/server.js [hunk 2]
```diff
@@ -9,6 +10,9 @@ const app = express();
 app.use(morgan(logLevel));
 app.use(express.json());
 app.use('/v1', v1Router);
+app.use(cors({
+  origin: 'http://localhost:3001',
```
Comment (JakeBent) [inline src/server.js]: store the string as a constant called `corsHost` in ` config/index.js` similar to `dbUrl`, `port`, etc.
No hardcoded strings in the `server.js` file, we need them all to be overwrite-able via an environment variable.

**Review by JakeBent — CHANGES_REQUESTED** [review]
minor changes, almost there.

**Review by JakeBent — COMMENTED** [review]
LGTM

**Review by JakeBent — APPROVED** [review]

**Pull request merged**